01_host/05_consensus/block_production.adoc

```diff
@@ -94,6 +95,12 @@ build a block. The stem:[sk] is the block producer lottery secret key and
 stem:[n] is the index of the epoch for whose slots the block producer is running
 the lottery.
 
+In order to ensure consistent block production, BABE uses secondary slots in
+case no authority won the (primary) block production lottery. Unlike the
+lottery, secondary slot assignees are know upfront publically
+(<<block-production-secondary-slots>>). The Runtime dictates what types of
+rounds to execute (<<sect-rte-babeapi-epoch>>).
```

Comment:
> The Runtime dictates what types of rounds to execute
I am not sure what that could mean. If you mean that the runtime provides information about what slots are currently in use through the babe configuration and consensus messages, it would be best just to link to them here.